packages/vanilla/src/types.ts

```diff
@@ -51,6 +51,7 @@ export type OverlapMode = 'keep' | 'drop' | 'invert';
 export interface Scrolling {
     speedDivider: number;
     manualSpeed: number;
+    startScrollMargins: { x: number, y: number }
```

Comment:
Could you add the `"object-curly-spacing": ["error", "never"]` rule to the root eslint config? Otherwise there'll be inconsistencies in formatting :)